### DISCUSSION TITLE
Warning: Undefined array key

### DISCUSSION THREAD

**Discussion by taylord2020** [code4recovery/12-step-meeting-list#1025] [Q&A] [answered]
Hi Josh and Dev Team...

Our site is throwing the error messages below. It was first noticed today.

It's probably something simple or obvious, but I can't figure it out.

* https://santabarbaraaa.com/
* TSML Version 3.14.9
* Wordpress version: 6.1.1
* I tried deactivating all plugins except TSML but that the errors persisted

![image](https://user-images.githubusercontent.com/62483893/210024628-d25bc065-543d-4cc9-8b99-bef705d3b024.png)

**Comment by joshreisner** [comment, marked as answer]
hm! sounds like you have a meeting without a time and WP_DEBUG mode set to on, so its complaining. not sure why it's not rendering the whole list though! 

we can fix that on our end, but in the meantime you might want to check if you need that appointment meeting. looks like the meeting in question is https://santabarbaraaa.com/meetings/sb-aa-convention-planning-meeting - could you try removing that? 

another thing you can do to hide the warning is set `WP_DEBUG` to `false` in your `wp-config.php`

finally you could try switching the user interface to TSML UI

**Reply by taylord2020** [reply]
Edit: I think the reason it's not findable with the TSML UI search is because the status is stuck in Draft mode. I'll keep playing around with this, but it's good enough for now. Thanks again.

Hey Josh,

I first deleted the meeting and that worked; No errors.

Then I switched to the TSML UI, and restored the meeting. Still no errors, but the meeting doesn't show up in search (see cap below). I'm concerned other meetings might also not be findable so I'm going to switch back the the Legacy UI, and find out if they intended to set it up as an "appointment" meeting.

I really appreciate your service! Happy New Year!

![image](https://user-images.githubusercontent.com/62483893/210032196-27f34449-d53b-4bad-b519-d518ead6e364.png)

**Reply by joshreisner** [reply]
Thanks for letting us know, we've got a fix coming soon #1027 that should fix the issue you were having.

This is sort of beside the point about the error, but just wanted to mention that the AA convention planning meeting is not a great fit for TSML because it's not a weekly recovery meeting - we recommend using an events calendar or other page for anything that's not a weekly recovery meeting.

**Reply by taylord2020** [reply]
After I spoke with Central Office yesterday, they decided to remove it from the meeting list, and only list regular weekly meetings like you suggested. Thanks for all you help with this.
